maple-for-lenders/defaults-and-impairments.md

```diff
@@ -2,16 +2,16 @@
 
 **What is a default?**
 
-Defaults can be executed by a pool delegate when a borrower has not made a payment (including non-payment or insufficient payment of principle, interest, or late fees as applicable) past the grace period on a loan. A default will:
+Defaults can be executed by a Pool Delegate when a borrower has not made a payment (including non-payment or insufficient payment of principal, interest, or late fees as applicable) past the grace period on a loan. A default will:
 
 1. Reduce the pool’s value by the amount of outstanding principle on the loan and any interest accrued.
 2. Sell any collateral on the loan and increase the value of the pool to the value of the liquidated collateral.
-3. Move Delegate first loss capital to the value of the liquidation max per default into the pool.
+3. Move Delegate first loss capital (if applicable) to the value of the liquidation max per default into the pool.
 
 **What is an Impairment?**
 
-The Maple platform gives Pool Delegates the ability to Impair loans in the event of a potential non-payment or insufficient payment scenario, or in a scenario where a borrower has met another condition of default as articulated in the loan agreement. The impairment can be put in place to better distribute potential losses to all current lenders. Impairment prevents a situation where a loan is known to be compromised (by borrower distress or otherwise), and some lenders withdraw their capital prior to the loss, leading to the remaining lenders shouldering more of the burden of the lost capital. This may be done in a situation where the Pool Delegate believes they will be able to quickly recover a part or all of the loan’s value from the Borrower through collaborative efforts or restructuring. The Pool Delegate will maintain an active dialogue with the Borrower to recover funds through the legal process.
+The Maple platform gives Pool Delegates the ability to impair loans in the event of a potential non-payment or insufficient payment scenario, or in a scenario where a borrower has met another condition of default as articulated in the loan agreement. Impairments were designed to be used in a scenario where the Borrower is not yet in a technical default period but Maple has been made aware that the Borrower may not be able to pay their loan in full. The impairment can be put in place to better distribute potential losses to all current lenders. Impairment prevents a situation where a loan is known to be compromised (by borrower distress or otherwise), and some lenders withdraw their capital prior to the loss, leading to the remaining lenders shouldering more of the burden of the lost capital. This may be done in a situation where the Pool Delegate believes they will be able to quickly recover a part or all of the loan’s value from the Borrower through collaborative efforts or restructuring. The Pool Delegate will maintain an active dialog with the Borrower to recover funds through the legal process.
```

Comment:
`Impairments were designed`  -> remove the past tense aspect of this

`otherwise), and some` -> do we need the comma here?